src/hooks/use-import-student-gradebook-data.ts

```diff
@@ -31,8 +31,16 @@ export const useImportStudentGradebookData = () => {
     },
   })
 
+  const importGradebookDataWithToast = async() => {
+    toast({
+      title: "Importing student gradebook data",
+      description: "Please wait while we import student gradebook data from your LMS.",
+    })
+    await mutation.mutateAsync()
+  }
+
   return {
-    importStudentGradebookDataAsync: mutation.mutateAsync,
+    importGradebookDataWithToast,
```

Comment:
I kinda dislike the "with toast" part of the naming here, it feels unneeded? It had a toast before too? Knowing that the function is async from the name is useful, knowing it has a toast adds far less

Also we should just put it on line 11 before we return the defaultMutation. That would be the start of the flow.

Reply:
This applies to the other one too